docs/en/observability/metrics-app-fields.asciidoc

```diff
@@ -238,7 +238,7 @@ These fields must be mapped to display RDS database data in the {metrics-app}.
 
 `aws.rds.db_instance.arn`::
 
-Amazon Resource Name(ARN) for each rds.
+Amazon Resource Name(ARN) for each `rds`.
```

Comment:
```suggestion
Amazon Resource Name (ARN) for each RDS.
```

Reply:
I feel like this description is missing something, but I don't think "rds" belongs in backticks.